renderer/main.js

```diff
@@ -94,15 +95,18 @@ function onState (err, _state) {
   // ...window visibility state.
   document.addEventListener('webkitvisibilitychange', onVisibilityChange)
 
-  sound.play('STARTUP')
+  // Log uncaught JS errors
+  window.addEventListener('error', (err) => telemetry.logUncaughtError('window', err.error), true)
```

Comment:
Why are you setting the third argument to `window.addEventListener` to `true`? Was that intentional? It's setting `useCapture` which is pretty obscure: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener